src/main/scala/com/normation/rundeck/plugin/resources/rudder/DataModel.scala

```diff
@@ -94,6 +97,108 @@ final case class Configuration(
 //Rudder node ID, used as key to synchro nodes
 final case class NodeId(value: String)
 
+case class Data(
+    nodes: Seq[Node]
+)
+
+object Data {
+  given JsonDecoder[Data] = DeriveJsonDecoder.gen
+}
```

Comment:
This would be much more concise with :
```suggestion
case class Data(nodes: Seq[Node]) derives JsonDecoder
```